CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - setup: Automate process of building new system images
 - spine: Print out configuration dictionary upon reset
 - bazelisk: Add support for ARM64 CPUs
+- spine : Add extra URDFs position and orientation
```

Comment:
```suggestion
- spines : Expose environment body poses in Bullet spine
```

- body rather than URDF because URDF is the file format
- pose = position & orientation (shorter)